aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -220,6 +220,37 @@ If the custom `WebHostService` code requires a service from dependency injection
 
 Services that interact with requests from the Internet or a corporate network and are behind a proxy or load balancer might require additional configuration. For more information, see <xref:host-and-deploy/proxy-load-balancer>.
 
+## Configure HTTPS
+
+Specify a [Kestrel server HTTPS endpoint configuration](xref:fundamentals/servers/kestrel#endpoint-configuration). The following example specifies an HTTPS port with an [X.509 certificate (X509Certificate2)](/dotnet/api/system.security.cryptography.x509certificates.x509certificate2) provided by [CertificateLoader.LoadFromStoreCert](/dotnet/api/microsoft.aspnetcore.server.kestrel.https.internal.certificateloader.loadfromstorecert). The endpoint configuration:
+
+* Sets the port to 5001.
+* Uses the [local development certificate (localhost)](xref:aspnetcore-2.1#on-by-default).
```

Comment:
This is most likely the IIS Express development certificate, not the ASP.NET Core one unless you are manually moving the cert into the local store which you shouldn't be doing

Reply:
We can leave the cert configuration details in the kestrel docs and link there for more details.

Reply:
I had the app tell me what it was using ...

```
IIS Express Development Certificate
CN=localhost
```

>  link there

Let's do that and move on.